charts/fybrik/templates/katalog-connector/katalog-connector-rbac.yaml

```diff
@@ -1,18 +1,25 @@
 {{- $autoFlag := and .Values.coordinator.enabled (eq .Values.coordinator.catalog "katalog") }}
 {{- if include "fybrik.isEnabled" (tuple .Values.katalogConnector.enabled $autoFlag) }}
-{{- if .Values.clusterScoped }}
 # Grant katalog-connector the katalog-editor Role.
-kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.clusterScoped }}
+kind: ClusterRoleBinding
 metadata:
-  name:  katalog-connector-editor
+  name:  katalog-connector-viewer
```

Comment:
samples/operate-first/ also should be changed

Reply:
Thanks. Since samples/operate-first is already outdated I'll keep it off this PR (almost all CRDs are old). A separate PR to re-generate can be made by @eletonia if this will be required but currently it seems like we will only need CRDs as cluster scoped resources in operate-first.